### ISSUE TITLE
Responsible organisations tooltip accessibility

### ISSUE THREAD

**Issue by terotik** [kausaltech/kausal-watch-ui#36]
Tooltip descriptions on responsible organisations should be visible also when not hovered.
They should be read to screenreader or focused on with keyboard navigation.

Alternatively use organisation full name on tags

WCAG 2.1  1.4.13  AA
WCAG 2.1  2.1.1  A
WCAG 2.1  2.4.4  A
